src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: 300ms;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties